testing/sechecker.ini

```diff
@@ -254,6 +254,7 @@ exempt_source = arpwatch_t
                 drbd_t
                 fcoemon_t
                 firewalld_t
+                haproxy_t
```

Comment:
Please put a comment for why/how haproxy needs net_admin.

Reply:
I found that haproxy is still functioning in my setup if I `dontaudit` the access, so this won't be needed anymore. :)